.github/ISSUE_TEMPLATE/git-failure-due-to-authentication.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Git failure due to authentication
+about: Create a report to help us improve
```

Comment:
What does this line do? Let's remove it, it sounds awkward.

Reply:
The frontmatter gets turned into an issue template description.

![image](https://user-images.githubusercontent.com/5658207/51742231-0f446c80-2091-11e9-9f70-656164e42e54.png)

![image](https://user-images.githubusercontent.com/5658207/51742208-ffc52380-2090-11e9-9b5c-cd002d4b1624.png)

Reply:
That's correct.  

While we could also make a Documentation /Feature request issue, I think it's best to start with connectivity (the one that has the most questions) and go from there.  Note users can always bypass the template by clicking "Open a regular issue"

Reply:
Got it. This is fine then, if it's normal boilerplat.